### PR TITLE
Add more documentation to `ledger-databases` extension

### PR DESCRIPTION
**Pull request by PotatoPresident**
Add translations.md
Change `time` parameter docs

### CODE REVIEW

File: docs/parameters.md [hunk 1]
```diff
@@ -75,4 +75,9 @@ Example - `after:3h`
...
 It will select actions within the duration specified.
 You can specify durations with `s`econd, `m`inute, `h`our, `d`ay and `w`eek.
-You can also combine multiple durations for example `7w4d31m42s`. The `before` parameter selects all results before the point in time that was the provided, and after that point for `after`
\ No newline at end of file
+You can also combine multiple durations for example `7w4d31m42s`.
+The `after` parameter is the same as the `time` parameter in older versions.
+The `before` parameter selects all results before the point in time that was the provided, and after that point for `after`.
+An easy way to remember the difference between `before:1d` and `after:1d` is to think about it like this.
+If you go back in time 1 day, do you want everything that happened `before` then or `after` then.
+Usually you want `before`.
```
Comment (yitzy299) [inline docs/parameters.md:83]: Not really - mostly you rollback time from X up until now

**Review by yitzy299 — CHANGES_REQUESTED** [review]

File: docs/parameters.md (hunk 1 above)
Comment (PotatoPresident) [reply docs/parameters.md:83]: oh, yeah I meant `after`

File: docs/translations.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Translations
+Ledger currently supports many languages.
+The language used is based on the clients selected language and defaults to `en_us` if it doesn't support the clients language.
+
+### Supported Languages
+- `en_us`
+- `en_gb`
```
[the remark this answers is not in the document]
Comment (PotatoPresident) [reply docs/translations.md:7]: but its still there 🤔

[the remark this answers is not in the document]
Comment (yitzy299) [reply docs/translations.md:7]: My bad - it's removed in #96

**Review by yitzy299 — APPROVED** [review]

**Pull request merged**